sdks/java/harness/jmh/src/main/java/org/apache/beam/fn/harness/jmh/ProcessBundleBenchmark.java

```diff
@@ -185,23 +184,21 @@ public void log(LogEntry entry) {
     }
 
     @TearDown
-    public void tearDown() throws Exception {
-      controlServer.close();
-      stateServer.close();
-      dataServer.close();
-      loggingServer.close();
-      controlClient.close();
-      sdkHarnessExecutor.shutdownNow();
-      serverExecutor.shutdownNow();
+    public void tearDown() {
       try {
+        controlServer.close();
+        stateServer.close();
+        dataServer.close();
+        loggingServer.close();
+        controlClient.close();
         sdkHarnessExecutorFuture.get();
-      } catch (ExecutionException e) {
-        if (e.getCause() instanceof RuntimeException
-            && e.getCause().getCause() instanceof InterruptedException) {
-          // expected
-        } else {
-          throw e;
-        }
+      } catch (InterruptedException ignored) {
+        // expected
```

Comment:
Probably good practice to ` Thread.currentThread().interrupt();` here

Reply:
Fixing... thank you.